src/components/circular-progress/examples/circular-progress-props.tsx

```diff
@@ -43,6 +48,13 @@ export class CircularProgressPropsExample {
                 maxValue={this.maxStars}
                 suffix={this.stars}
             />,
+            <limel-circular-progress
+                size="x-large"
```

Comment:
not a big deal, but I'd say it's better that all have the same size

Reply:
it was only for testing purposes 😁 (easier to see) , I will remove it when it's done :)

Reply:
I added it as a fixup ;) made all of them big. That's what I meant. :)